### PR TITLE
Composer: update autoloader for latest AMP plugin changes

### LINKED ISSUE #7336: Composer: update autoloader for latest AMP plugin changes
## Bug Description

While testing another PR #7292, I updated AMP Plugin and AMP toolkit. After doing this, I started getting error in unit test. 
I locked the commit down composer file to https://github.com/ampproject/amp-toolbox-php/commit/8e1dfacf4f4df8835c6a9ed1d3be8b4a7bf7b92e and https://github.com/ampproject/amp-wp/commit/1bac57aba4e3b58351381b4967ab6ee9b9b00aa6 that this seem to fix this issue. 

## Screenshots
Error in PHPUnit
Screenshot 2021-04-29 at 18 01 32
After commenting out the line above. 
Screenshot 2021-04-29 at 18 03 00

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Update composer file and upgrade min required 5.6.20, to match WordPress core. 

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7336

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
   "homepage": "https://github.com/google/web-stories-wp",
   "license": "Apache-2.0",
   "require": {
-    "php": "^5.6 || ^7.0 || ^8.0",
+    "php": "^5.6.20 || ^7.0 || ^8.0",
```
Comment (swissspidy) [inline composer.json:8]: Why was this needed? Or did you just bump this while you were at it?

File: composer.json [hunk 2]
```diff
@@ -67,6 +67,7 @@
       "ampproject/amp-wp/back-compat/back-compat.php",
       "ampproject/amp-wp/includes/amp-helper-functions.php",
       "ampproject/amp-wp/includes/admin/functions.php",
+      "ampproject/amp-wp/includes/bootstrap.php",
```
Comment (swissspidy) [inline composer.json]: All other entries here need to be removed, except for this one

File: composer.json (hunk 1 above)
Comment (spacedmonkey) [reply composer.json:8]: I was seeing terminal. 
```
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires sabberworm/php-css-parser dev-master#bfdd976 -> satisfiable by sabberworm/php-css-parser[dev-master].
    - sabberworm/php-css-parser dev-master requires php >=5.6.20 -> your php version (5.6; overridden via config.platform, actual: 7.4.15) does not satisfy that requirement.
  Problem 2
    - sabberworm/php-css-parser dev-master requires php >=5.6.20 -> your php version (5.6; overridden via config.platform, actual: 7.4.15) does not satisfy that requirement.
    - ampproject/amp-wp dev-develop requires sabberworm/php-css-parser dev-master#bfdd976 -> satisfiable by sabberworm/php-css-parser[dev-master].
    - Root composer.json requires ampproject/amp-wp dev-develop -> satisfiable by ampproject/amp-wp[dev-develop].
```

Comment (spacedmonkey) [reply composer.json:8]: Also worth noting that 5.6.20, is min required for WordPress. 
https://github.com/WordPress/wordpress-develop/blob/aa62fdbfaad20eadf10ddad11d4be0279e6cb740/src/wp-includes/version.php#L37

Comment (swissspidy) [reply composer.json:8]: Ah, looks like a recent addition: https://github.com/sabberworm/PHP-CSS-Parser/pull/215

All good then 👍

**Review by swissspidy — APPROVED** [review]
